### PR TITLE
HelloWorld application using flask in a Vagrant enviroment

### PR DESCRIPTION
**Pull request by TzachiGitHub**
runs on port 5000 on the localhost of the user.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,7 @@
-# superx
+superx
```
Comment (simzacks) [inline README.md]: this file was already pushed.
You need to the existing repo so you don't have a conflict.

File: Vagrantfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+Vagrant.configure("2") do |config|
+  config.vm.box = "ubuntu/trusty64"	  
+  # Flask
+  config.vm.network "forwarded_port", guest: 5000, host: 5000
+  # config.vm.network "private_network", ip: "192.168.33.10"
```
Comment (simzacks) [inline Vagrantfile]: remove the commented out line

File: Vagrantfile [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+Vagrant.configure("2") do |config|
+  config.vm.box = "ubuntu/trusty64"	  
+  # Flask
+  config.vm.network "forwarded_port", guest: 5000, host: 5000
+  # config.vm.network "private_network", ip: "192.168.33.10"
+  config.vm.provider "virtualbox" do |vb|
+vb.name = "superx"
+end
+config.vm.provision "shell", path: "bootstrap.sh", privileged: false
+end
```
Comment (simzacks) [inline Vagrantfile]: end file with blank line

File: app.py [hunk 4]
```diff
@@ -0,0 +1,8 @@
+from flask import Flask
+app = Flask(__name__)
+
+@app.route('/')
+def hello():
+        return '<h1>Hello World!</h1>'
+
+app.run(host='0.0.0.0', port=5000)
```
Comment (simzacks) [inline app.py]: end file with blank line

**Review by simzacks — CHANGES_REQUESTED** [review]
Minor comments before getting started.
In general, we don't leave commented out code in committed code. 
If there's a reason to leave it in, there should be a comment explaining why it is being left in.

Other than these minors nits, the code is ready to be merged

File: README.md (hunk 1 above)
Comment (lmilbaum) [reply README.md]: See the discussion in Slack on how to update your local+fork repos with changes landed in head repo

File: README.md [hunk 5]
```diff
@@ -1 +1,7 @@
...
+Team members:
```
Comment (lmilbaum) [inline README.md]: How is this change related to the PR description. Do NOT mix few efforts in one PR.

File: Vagrantfile [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+# vi: set ft=ruby :
+Vagrant.configure("2") do |config|
+  config.vm.box = "ubuntu/trusty64"	  
+  # Flask
+  config.vm.network "forwarded_port", guest: 5000, host: 5000
+  # config.vm.network "private_network", ip: "192.168.33.10"
+  config.vm.provider "virtualbox" do |vb|
+vb.name = "superx"
+end
+config.vm.provision "shell", path: "bootstrap.sh", privileged: false
```
Comment (lmilbaum) [inline Vagrantfile]: Why are you using the `privileged: false` ?

File: app.py [hunk 7]
```diff
@@ -0,0 +1,8 @@
+from flask import Flask
+app = Flask(__name__)
+
+@app.route('/')
+def hello():
+        return '<h1>Hello World!</h1>'
```
Comment (lmilbaum) [inline app.py]: Indention seems off

File: bootstrap.sh [hunk 8]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+sudo apt-get update
```
Comment (lmilbaum) [inline bootstrap.sh]: sudo should not be used in a provisioning script

File: bootstrap.sh [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+sudo apt-get install python3
+sudo apt-get install python3-pip -y
```
Comment (lmilbaum) [inline bootstrap.sh]: Those two lines can be merged to one command

**Review by lmilbaum — CHANGES_REQUESTED** [review]

File: Vagrantfile (hunk 6 above)
Comment (TzachiGitHub) [reply Vagrantfile]: to run the bootstrap.sh file without admin priviledges, 
so if I try something I shouldn't without the 'sudo' command - it will let me know and stop me.
for future programming.

**Review by TzachiGitHub — COMMENTED** [review]
I've followed the comments and updated the files as requested.

**Review by simzacks — APPROVED** [review]
